### ISSUE TITLE
Is this suitable for stand-alone web applications?

### ISSUE THREAD

**Issue by nikeee** [microsoft/vscode-webview-ui-toolkit#294]
With some hacks (providing a CSS with the respective variables extracted from VSCode), these controls can be used for stand-alone web applications outside of VSCode.

It seems that it's not currently in the scope. Is this planned?

**Comment by hawkticehurst** [comment]
Hey @nikeee, from a technical perspective the toolkit components should theoretically work fine in a stand-alone web app since they're just web components.

Also as you noted, yes, you will probably need to provide the correct CSS variables if you want proper theming. However, the toolkit components ship with the VS Code dark theme as a default out of box.

With that said, we've built this library with VS Code's webview extension environment specifically in mind (which can sometimes be slightly different from a traditional browser environment [we've discovered](https://github.com/microsoft/vscode-webview-ui-toolkit/issues/193)), so as maintainers we cannot make any strong guarantees about the toolkit's suitability in web apps since we're not building or testing for that environment.
